lib/jekyll/tags/strapiimagefilter.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+require "down"
+
+module Jekyll
+      class StrapiStaticFile < StaticFile
+        def initialize(site, base, dir, name, collection = nil)
+          @site = site
+          @base = base
+          @dir  = dir
+          @name = name
+          @collection = collection
+          # TODO: Check if user can change 'assets' path
+          @relative_path = File.join(*["assets", @name].compact)
+          @extname = File.extname(@name)
+        end
+      end
+
+    module StrapiImageFilter
+      def asset_url(input)
+        assets_path = "assets" # TODO: to figure our in which conditions it can change
+        strapi_endpoint = @context.registers[:site].config['strapi']['endpoint']
+        uri_path = "#{strapi_endpoint}#{input['url']}"
```

Comment:
Where is **url** being created? I don't see any other reference of url.

I see this line: but no url that is defined. uri_path = "#{strapi_endpoint}#{input['url']}"

Reply:
https://api.example.com/api/photos/?populate=* is where the url is being referenced. I have this at:  ```"/uploads/quantum_qt5_ee7a744bdd.webp"```